### PR TITLE
E2E tests for external cluster functionality

### PR DESCRIPTION
**Pull request by maciaszczykm**
### Which issue(s) this PR fixes
Fixes https://github.com/kubermatic/dashboard/issues/3309.
Closes https://github.com/kubermatic/dashboard/issues/3298.

### Release note
```release-note
NONE
```

### CODE REVIEW

**Comment by maciaszczykm** [comment]
Both tests failed on cluster deletion.

/retest

**Comment by maciaszczykm** [comment]
/hold

Unfortunately, current kubeconfig doesn't work. Also, minio may need some changes.

**Comment by maciaszczykm** [comment]
/hold cancel

@kgroschoff PTAL

File: cypress/pages/clusters.po.ts [hunk 1]
```diff
@@ -34,6 +53,10 @@ export class ClustersPage {
     return cy.get('#km-delete-cluster-btn');
   }
 
+  static getDiscnnectConfirmBtn(): Cypress.Chainable {
```
Comment (kgroschoff) [inline cypress/pages/clusters.po.ts:56]: NIT: "disconnect" it missing the o

**Review by kgroschoff — COMMENTED** [review]
just a very very small remark. apart from that LGTM

File: cypress/pages/clusters.po.ts (hunk 1 above)
Comment (maciaszczykm) [reply cypress/pages/clusters.po.ts:56]: Done, PTAL.

**Review by kgroschoff — COMMENTED** [review]
/lgtm

**Pull request merged**